cmd/deploy.go

```diff
@@ -152,11 +161,12 @@ func InitRunCommand() (*cobra.Command, error) {
 			concurrencyDefault = 0
 		}
 	}
-	var concurrency int
-	run.Flags().IntVarP(&concurrency, "concurrency", "c", concurrencyDefault, "concurrency")
+	run.Flags().IntP("concurrency", "c", concurrencyDefault, "concurrency")
 
-	var clusterUrl string
-	run.Flags().StringVar(&clusterUrl, "url", os.Getenv("KUBERNETES_CLUSTER_URL"),
+	run.Flags().String("url", os.Getenv("KUBERNETES_CLUSTER_URL"),
 		"URL of the Kubernetes cluster. Overrides KUBERNETES_CLUSTER_URL env variable in AppController pod.")
+
+	run.Flags().StringP("replicas", "n", "1", "number of flow replicas: [+]number")
```

Comment:
OK, I get it now. But I think that this flag should have a long and descriptive name instead of `n`, like `--replica-count` or `--desired-replicas`. `n` is not enough.